user-guide/Advanced_Modules/DOM/DOM_data_storage.md

```diff
@@ -14,3 +14,12 @@ All DOM data are stored in the Elasticsearch database. Data for each object are
 | [DomInstance](xref:DomInstance) | cdominstance_{moduleId} |
 | [DomBehaviorDefinition](xref:DomBehaviorDefinition) | cdombehaviordefinition_{moduleId} |
 | [HistoryChange](xref:DOM_history#historychange) | chistory_dominstance_{moduleId} |
+
+## Caching
+
+By default, a full cache of the DOM configuration objects (`SectionDefinition`, `DomDefinition`, and `DomBehaviorDefinition`) is present. This means that all of these objects are stored in memory to improve the performance of the DOM manager. These caches are kept in sync across the cluster using the [DOM events](xref:DOM_events).
+
+> [!NOTE]
+>
+> - Correct delivery of the DOM events is important to keep these caches in sync. If the DMS is unstable and Agents frequently lose connection to each other, we recommend disabling the caches. You can do so using the [caching settings](xref:DOM_StorageSettings#cachingsettings) in the `ModuleSettings`.
+> - If a DOM configuration object event is lost, the cache may not reflect the correct state. This will be automatically fixed at midnight, as the "midnight sync" will ensure that the caches are correct by comparing them with the database. You can also trigger this sync manually in the [SLNetClientTest tool](xref:Opening_the_SLNetClientTest_tool) on the maintenance tab of a DOM manager viewer window. (Advanced > Apps > DataMiner Object Model)
```

Comment:
I'd prefer if the information about the SLNetClientTest tool sync were added in more detail on a separate page, similar to [Removing a DOM manager](https://docs.dataminer.services/user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_removing_DOM_Manager.html). Right now this doesn't really describe how you trigger the sync, only where to go in the tool. If this is moved to a separate page, there will be more room to give this explanation all the necessary details, and then the standard warning about SLNetClientTest tool can also be included.